flatdata-rs/lib/src/storage.rs

```diff
@@ -341,12 +341,6 @@ impl<'a> ResourceHandle<'a> {
     }
 }
 
-impl<'a> Drop for ResourceHandle<'a> {
-    fn drop(&mut self) {
-        panic!("Resource not closed");
```

Comment:
Do we really want to remove it? Before we had a check here, if we are not panic'ing already, which seemed to work.

Reply:
Yeah, this was doing something interesting: When another panic happened, during stack unrolling this was panicking and killing the app.

Also matches what we discussed in #66

Reply:
How about still trying to close the resource in Drop? And if fails, catch the error, log it and not to do anything anymore.

Reply:
Good idea, will try to do that (also in multivector/externalvector)

Reply:
Since this seems more complicated (especially in MultiVector it is hard to do without performance impact), I reverted to the previous behaviour for now.